spaces/domains.json

```
@@ -1,6 +1,7 @@
 {
   "dao.anuinitiative.org": "daotest.anuinitiative.eth",
   "dao.chaingpt.org": "chaingptai.eth",
+   "dao.ampleswap.com": "ampleswap.eth",
```

Please remove extra space before your line.
```suggestion
  "dao.ampleswap.com": "ampleswap.eth",
```